docs/modules/blocks/enderchests.mdx

```diff
@@ -157,7 +156,7 @@ Additionally, players cannot place Ender chests under any circumstances.
           The region the drop-off applies to.
         </td>
         <td>
-          <a href="/docs/modules/mechanics/regions">Bounded Regions</a>
+          <a href="/docs/modules/mechanics/regions">Bounded Region</a>
```

Comment:
given that the region & filter are defined as properties, there's no reason to re-repeat the same thing in a second table. the whole "Dropoff Sub-elements" table can go

Reply:
This is mostly precedent from other pages having a similar layout. [Block Drops](https://pgm.dev/docs/modules/blocks/blockdrops), [Falling Blocks](https://pgm.dev/docs/modules/blocks/falling-blocks), [Portals](https://pgm.dev/docs/modules/mechanics/portals)

Reply:
i thought the standard was to list them just in the attributes region with a "property" tag, if it seems more common to list them in both places then that's fine, but i think some other places will need expansion as not all attributes marked as property have another table for the children

Reply:
I'll remove it from the enderchest page for now and clean up the rest of the pages in a future pull request

Reply:
![image](https://github.com/PGMDev/Website/assets/20259871/5a169154-a669-4d69-8c50-30da470c3dcb)